doc/api/next_api_changes/behavior/21238-AL.rst

```diff
@@ -0,0 +1,6 @@
+``CallbackRegistry`` raises on unknown signals
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Callback registries instantiated by Matplotlib now check that callbacks are
```

Comment:
Should link to `CallbackRegistry` somewhere in the text, since it can't be done in the title.

Reply:
yes